ICA.md

```diff
@@ -290,22 +290,21 @@ relationship;
 
 6.4.2. Contractor, under no circumstances is entitled to receive from Customer
 any benefits or insurance, including without limitation any unemployment,
-workers compensation or disability benefits, vacation or sick pay; 
+worker's compensation or disability benefits, vacation or sick pay; 
 
 6.4.3. Contractor is solely responsible for determining the method and means
 by which Customer will accomplish the Services and otherwise fulfill
 Contractor's obligations hereunder; and 
 
 6.4.4. Contractor will be solely responsible for the professional performance
-of the Services, and will receive no assistance, direction or control from
+of the Services, and will receive no assistance, direction, supervision or control from
 Customer except as specifically set forth in Standard Conditions. 
 
+<!-- This is contradictory to the point 6.4.2 - it states that you won't pay for insurance, unemployement, worker's comp etc. but here states that you will pay for the insurance costs for all of these. Additionally, as you don't want to set up an employment contract, you might want to get rid of the 'worker's compensation' reference  -->
 6.5. Contractor shall be solely responsible for obtaining, at Customer's
 expense and in Contractor's name, disability, worker's compensation or other
 insurance as well as all licenses and permits usual or necessary for
-conducting the Services hereunder. Contractor represents that Contractor has
-the qualifications, skills, and ability to perform the Services in a
-professional manner, without the advice, control or supervision of Customer;
+conducting the Services hereunder. 
```

Comment:
I've merged this into part 6.4.4, which was largely the same.

Reply:
You're right, this clause should be removed entirely. The only expense that shall be passed onto the customer is agreed-upon expenses. Things like project-specific equipment.